config.go

```diff
@@ -11,12 +11,18 @@ import (
 
 func config() (Bin, mqttConfig) {
 	var mqttServer string
+	var mqttUser string
+	var mqttPassword string
+	var mqttStateTopic string
 	var sensorName string
 	var binFullTime float64
 	var unitOfMeasurement string
 	var FilePath string
 	var LoggingLevel string
 	flag.StringVar(&mqttServer, "mqtt_server", "mqtt://localhost:1883", "mqtt broker address")
+	flag.StringVar(&mqttUser, "mqtt_user", "", "mqtt user")
```

Comment:
Can you modify the mqtt_user and mqtt_password flags to lookup the values from the corresponding environment variables?

```go
flag.StringVar(&mqttUser, "mqtt_user", "", "mqtt user")
flag.StringVar(&mqttPassword, "mqtt_password", "", "mqtt password")
```
```go
flag.StringVar(&mqttUser, "mqtt_user", lookUpEnv("MQTT_USER", ""), "mqtt user")
flag.StringVar(&mqttPassword, "mqtt_password", lookUpEnv("MQTT_PASSWORD", ""), "mqtt password")
```
```go
func lookUpEnv(variable, defaultVariable string) string {
	if envVariable, ok := os.LookupEnv(variable); ok {
		return envVariable
	}
	return defaultVariable
}
```

Reply:
I've updated the repo. Is it okay like this?

Reply:
Yeah, looks good. I will merge it and create another release for it later today. Thanks for the PR.